clang/lib/Driver/ToolChains/Flang.cpp

```diff
@@ -203,6 +203,30 @@ void Flang::AddAArch64TargetArgs(const ArgList &Args,
   }
 }
 
+void Flang::AddPPCTargetArgs(const ArgList &Args,
+                             ArgStringList &CmdArgs) const {
+  bool VecExtabi = false;
+  for (const Arg *A : Args.filtered(options::OPT_mabi_EQ)) {
```

Comment:
Could the loop be simplified to something like this?
```
Arg *A = Args.getLastArg(options::OPT_mabi_EQ);
if (A) {
  StringRef V = A->getValue();
  if (V == "vec-default") {
 [...]
}
```

Reply:
Will do